data-report-commons/src/test/java/commons/db/GraphStoreProtocolConnectionTest.java

```diff
@@ -31,7 +31,7 @@ class GraphStoreProtocolConnectionTest {
     @BeforeAll
     static void setUp() {
         var dataSet = DatasetFactory.createTxnMem();
-        initializeGraphServer(dataSet);
+        server = FusekiTestingServer.init(dataSet, GSP_ENDPOINT);
```

Comment:
Question: Why is GSP_ENDPOINT not an env var?

Reply:
It doesn't vary depending on environment, does it? Maybe it should be some common application constant?